test/jdk/jdk/jfr/event/gc/objectcount/ObjectCountEventVerifier.java

```diff
@@ -70,7 +70,7 @@ public static void verify(List<RecordedEvent> objectCountEvents) throws Exceptio
     private static long expectedFooArraySize(long count) {
         boolean runsOn32Bit = System.getProperty("sun.arch.data.model").equals("32");
         int bytesPerWord = runsOn32Bit ? 4 : 8;
-        int objectHeaderSize = bytesPerWord * 3; // length will be aligned on 64 bits
+        int objectHeaderSize = bytesPerWord * 2; // length will be in klass-gap
```

Comment:
Would it be in klass-gap on x86_32? Run this test on x86_32 to confirm?

Reply:
> Would it be in klass-gap on x86_32? Run this test on x86_32 to confirm?

Good catch! Indeed, on 32 bits, the length is in its own field, and thus we need a multiplier of 3. Fixed in a change that I just pushed, and re-ran the affected tests both on x86_32 and x86_64 for verification.

Reply:
Yes it looks good, my comment was off. Sorry for the noise :(